File: backend/waitingNow/src/main/java/com/waiting/waitingnow/domain/OptionMenuVO.java
package com.waiting.waitingnow.domain;

public class OptionMenuVO {
    private int optionMenuKey;
    private int menuNumber;
    private String optionMenuName;
    private int optionMenuPrice;
    private String optionMenuCategory;

    public int getOptionMenuKey() {
        return optionMenuKey;
    }

    public void setOptionMenuKey(int optionMenuKey) {
        this.optionMenuKey = optionMenuKey;
    }

    public int getMenuNumber() {
        return menuNumber;
    }

    public void setMenuNumber(int menuNumber) {
        this.menuNumber = menuNumber;
    }

    public int getOptionMenuPrice() {
        return optionMenuPrice;
    }

    public void setOptionMenuPrice(int optionMenuPrice) {
        this.optionMenuPrice = optionMenuPrice;
    }

    public String getOptionMenuName() {
        return optionMenuName;
    }

    public void setOptionMenuName(String optionMenuName) {
        this.optionMenuName = optionMenuName;
    }

    public String getOptionMenuCategory() {
        return optionMenuCategory;
    }

    public void setOptionMenuCategory(String optionMenuCategory) {
        this.optionMenuCategory = optionMenuCategory;
    }
}

File: flutter/waitingnow/lib/src/Domain/OptionMenuVO.dart
class OptionMenuVO {
   int? optionMenuKey;
   int? menuNumber;
   String? optionMenuName;
   int? optionMenuPrice;
   String? optionMenuCategory;

   OptionMenuVO.fromJson(Map<String, dynamic> json) {
      optionMenuKey = json['optionMenuKey'];
      menuNumber = json['menuNumber'];
      optionMenuName = json['optionMenuName'];
      optionMenuPrice = json['optionMenuPrice'];
      optionMenuCategory = json['optionMenuCategory'];
   }

   Map<String, dynamic> toJson() {
      return {
         'optionMenuKey': optionMenuKey,
         'menuNumber': menuNumber,
         'optionMenuName': optionMenuName,
         'optionMenuPrice': optionMenuPrice,
         'optionMenuCategory': optionMenuCategory,
      };
   }
}